.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
+https://github.com/checkpoint-restore/criu/blob/criu-dev/CONTRIBUTING.md
+
+In short you need to:
+
+- Describe What you do and How you do it;
+- Separate each logical change into a separate commit;
+- Add a "Signed-off-by:" line identifying that you certify your work with DCO;
+- If you fix some specific bug or commit, please add "Fixes: ..." line;
+- Review fixes should be ammended to the original commits, generally you would do something like:
```

Comment:
Review fixes should be made by amending the original commits. For example:

Reply:
done.

Reply:
Thanks 🙂 

Is the double colon in `For example::` intentional?

Reply:
Fixed. =)